### PR TITLE
feat:#512 Moved strategies to pages directory with i18n

### PR DESCRIPTION
**Pull request by Yingxue0323**
1. Added i18n for strategies page
2. Moved strategies to pages directory
3. Added config to store constants, fixed some relative paths

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -22,6 +22,6 @@
       "@/*": ["./*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "next.config.js"],
   "exclude": ["node_modules"]
```
Comment (ucfyao) [inline tsconfig.json:26]: What is the purpose of placing JavaScript files in the configuration of TS?

File: src/app/strategies/[strategyId]/page.tsx [hunk 2]
```diff
@@ -11,6 +11,7 @@ import Pagination from '../../../component/pagination';
 import '../../globals.scss';
```
Comment (ucfyao) [inline src/app/strategies/[strategyId]/page.tsx:11]: If possible, try to place it under the pages.

Comment (Yingxue0323) [reply src/app/strategies/[strategyId]/page.tsx:11]: > place it under the pages.

Due to the dynamic routing problem, we're still tring to figure out how to do it. Will do in another branch.

File: tsconfig.json (hunk 1 above)
Comment (Yingxue0323) [reply tsconfig.json:26]: > What is the purpose of placing JavaScript files in the configuration of TS?

Automatically added by compiler. 
There's still a small issue with next.config.js, will have a look after.

File: src/app/strategies/[strategyId]/page.tsx (hunk 2 above)
Comment (HuijunDev) [reply src/app/strategies/[strategyId]/page.tsx:11]: > If possible, try to place it under the pages.

I'll do this in another branch.

File: next.config.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 /** @type {import('next').NextConfig} */
-const { i18n } = require('./next-i18next.config.js');
+import pkg from './next-i18next.config.js';
+const { i18n } = pkg;
```
Comment (ucfyao) [inline next.config.js:4]: Please delete this configuration, we have already used other methods. Note that the front-end is the ES syntax.

**Review by ucfyao — APPROVED** [review]

**Pull request merged**